source/_integrations/nexia.markdown

```diff
@@ -74,3 +74,22 @@ The following binary sensors are added for each thermostat zone:
 ### Climate
 
 The `nexia` climate platform lets you control a thermostat.
+
+### Service `set_aircleaner_mode`
+
+Part of the `nexia.` services. Sets the air cleaner mode. Options include 'auto', 'quick', and 
+'allergy'. This setting will affect all zones on the same thermostat.
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id` | yes | String or list of strings that point at `entity_id`'s of climate devices to control. Else targets `nexia` thermostats.
```

Comment:
When using entity services, this becomes `no`, since a user can target an area as well.

```suggestion
| `entity_id` | no | String or list of strings that point at `entity_id`'s of climate devices to control. Else targets `nexia` thermostats.
```